opensubtitles/search.go

```diff
@@ -0,0 +1,78 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+//OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#search
+type SearchService service
+
+//SearchOptions contains the url parameter for the SearchService
+type SearchOptions struct {
+	//Title
+	Query string `url:"query,omitempty"`
+}
+
+//Movie search a movie by name
+//OpenSubtitles API docs : https://www.opensubtitles.com/docs/api/html/index.htm#search-for-a-movie
+func (s *SearchService) Movie(ctx context.Context, opt *SearchOptions) (shows *Shows, resp *http.Response, err error) {
+	u := "/api/v1/search/movie"
+	u, err = addOptions(u, opt)
+	if err != nil {
+		return nil, nil, nil
+	}
+	req, err := s.client.NewRequest("GET", u, "")
+	if err != nil {
+		return nil, nil, err
+	}
+
+	resp, err = s.client.Do(ctx, req, &shows)
+	if err != nil {
+		return nil, resp, err
+	}
+	return
```

Comment:
Codacy found an issue: [extra empty line at the end of a block](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)